modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-fido2-credential-service/src/main/resources/META-INF/sql/tables.sql

```diff
@@ -6,7 +6,8 @@ create table MFAFIDO2CredentialEntry (
 	userName VARCHAR(75) null,
 	createDate DATE null,
 	modifiedDate DATE null,
-	credentialKey VARCHAR(128) null,
+	credentialKey VARCHAR(75) null,
```

Comment:
hey @martamedio, 
per our discussion from yesterday this seems off, isn't it?
the type seems shorter than before now, so maybe something is missing.

Reply:
@csierra 
I was wondering about it as well, but looking at the other of code, you can see the the crednetialKey column is changed to LONGTXT type, and I checked the table after upgrading, it's indeed LONGTXT and is able to hold the key generated from softu2f.

Reply:
maybe from upgrade but when it is freshly created it will be off, wouldn't it be?

Reply:
Let me try it out

Reply:
As you pointed out: it should be a _LONGTEXT_, I don't know what instruction has changed this field but it's wrong